plugins/com.google.cloud.tools.eclipse.dataflow.ui.test/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTabTest.java

```diff
@@ -16,15 +16,83 @@
 
 package com.google.cloud.tools.eclipse.dataflow.ui.launcher;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.cloud.tools.eclipse.dataflow.core.project.MajorVersion;
+import com.google.cloud.tools.eclipse.test.util.ui.CompositeUtil;
+import com.google.cloud.tools.eclipse.test.util.ui.ShellTestResource;
+import com.google.common.base.Predicate;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Group;
+import org.eclipse.swt.widgets.Shell;
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+
 public class PipelineArgumentsTabTest {
 
+  @Rule public ShellTestResource shellResource = new ShellTestResource();
+
   private PipelineArgumentsTab tab = new PipelineArgumentsTab();
-  
+
   @Test
   public void testGetName() {
     Assert.assertEquals("Pipeline Arguments", tab.getName());
   }
 
+  @Test
+  public void testRunnerButtonChecked_majorVersionOne() {
```

Comment:
Is it worth making these a `Parameterized` test (subclass, run via `@RunWith(Suite.class)`; [example](https://github.com/apache/beam/blob/master/runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/CoderTranslationTest.java#L116)), in the style of the other `MajorVersion` tests? You could have a "TestSpec" or similar class which maps the MV to the expected default runner.